spec/features/donuts/01_user_can_view_a_list_of_donuts_spec.rb

```diff
@@ -11,26 +11,34 @@
 
   let!(:donut) { create(:donut) }
 
-  scenario "sees a list of donuts and a link for a new donut" do
+  scenario "an unauthenticated user is directed to the root and sees a
+    list of donuts" do
     visit root_path
 
-    expect(page).to have_content donut.name
-    expect(page).to have_link donut.name
+    expect(page).to have_content "Glazed"
+    expect(page).to_not have_link "Add New Donut"
+  end
 
-    click_link "Add New Donut"
+  scenario "an authenticated user is directed to root and sees a list of
+    donuts and a link to add a new donut" do
+    user = create(:user)
```

Comment:
This is used twice. I would bring this out into a let statement at the beginning. One way to do this is do wrap these scenarios in a context block `context 'as an autheticated user' do` will have a `let(:authenticated_user)` and then `scenario 'i will see a list` ... etc